### PR TITLE
[Merged by Bors] - Fix player number in example game in the ecs_guide

### PR DESCRIPTION
**Pull request by dilyankostov**
# Objective

- Small bug in the example game given in examples/ecs/ecs_guide

Currently, if there are 2 players in this example game, the function exclusive_player_system can add a player with the name "Player 2". However, the name should be "Player 3". This PR fixes this. I also add a message to inform that a new player has arrived in the mock game.

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]
Thanks!

File: examples/ecs/ecs_guide.rs [hunk 1]
```diff
@@ -211,9 +211,13 @@ fn exclusive_player_system(world: &mut World) {
     };
     // Randomly add a new player
     if should_add_player {
+        println!(
+            "An exclusive player {} has joined the game!",
```
Comment (Nilirad) [inline examples/ecs/ecs_guide.rs:215]:
```suggestion
            "Player {} has joined the game!",
```

The term “exclusive” in the function name refers to itself being an exclusive system. It doesn't have to do with the player.

**Review by Nilirad — CHANGES_REQUESTED** [review]
Nice bug busting here. Thanks 😀

I added a comment below with a small correction.

File: examples/ecs/ecs_guide.rs (hunk 1 above)
Comment (dilyankostov) [reply examples/ecs/ecs_guide.rs]: Good point, I've changed it

**Review by Nilirad — APPROVED** [review]

**Pull request closed without merging**